beast/physicsmodel/stars/tests/test_spectral_grid.py

```diff
@@ -28,6 +28,9 @@ def test_make_kurucz_tlusty_spectral_grid():
     # read in the cached isochrones
     oiso = ezIsoch(iso_fname)
 
+    # remove the isochrone points with logL=-9.999
+    oiso = ezIsoch(oiso.selectWhere("*", "logL > -9"))
+
```

Comment:
Note that this way of filtering the isochrones will not work once PR #553 is merged.  This relies on eztable specific functions.  New method (once #553 is merged) is: `oiso.data = oiso[oiso["logL"] > -9]`.